cybered/hashing/templates/hashing/motivation.html

```diff
@@ -0,0 +1,94 @@
+<!doctype HTML>
+<html>
+{% include "includes/head.html" %}
+
+<body>
+    {% include "includes/header.html" %}
+    <div class="container">
+
+        <h3>An Example</h3>
+        <p>
+            Alice wants to send a long, complicated email to Bob. She does not care if other people see the message, but
+            it is <i>very</i>
+            important that Bob receives the message <i>exactly</i> the same as she sends it. Chuck does not like Bob,
+            and he
+            tries to intercept the message and change it so that Bob gets the wrong message! What are some ways that
+            Alice and Bob
+            might try to make sure Bob gets the correct message?
+            <ul>
+                <li>Alice could tell Bob the message in-person, but that would be hard because it is a long message.
+                </li>
+                <li>Alice could encrypt the message, and then tell Bob the secret key for the encryption, but that would
+                    take a long time because it is a long message.</li>
+                <li>Alice could generate a message digest for her message, and tell that to Bob in person. Bob can use
```

Comment:
Make **message digest** bold?

Reply:
I tried to bold them at the point where they were defined; do you think it makes more sense to bold them everywhere, or all places before the definition instead?

Reply:
Hmm. I was thinking they should be bolded at their first use and at their definition. I'll leave it to your best judgement.